common/services/frameworks.js

```diff
@@ -144,6 +146,13 @@ const frameworksService = {
       field['ignore-defaults'] = true
       // use `default` to start with one empty item
       field.default = [{}]
+      // default minimum items to 1
+      field.minItems = minItems
```

Comment:
Had a quick chat with @cwrw and seeing as we don't really want to add it fully to the framework until we have actual use cases for needing custom minimum and maximum we thought let's just determine the min length based on the presence of required for now.

I think it's good to have the ability in the component but in this file we can do something simpler for now.

Reply:
Updated to take the default from the validations.